Makefile.am

```diff
@@ -278,7 +278,7 @@ AM_CPPFLAGS = \
 	-DSRCDIR=\"$(abs_srcdir)\" \
 	-DBUILDDIR=\"$(abs_builddir)\" \
 	-DDATADIR=\"$(datadir)\" \
-	-DPACKAGE_LIBEXEC_DIR=\""$(libexecdir)"\" \
+	-DLIBEXECDIR=\""$(libexecdir)"\" \
```

Comment:
What is the motivation for his change?

Reply:
I don't have a strong opinion about it, but it seems weird to rename the original variable in such an arbitrary way. We can also keep `PACKAGE_LIBEXEC_DIR` and just use that in this patch, of course.

Reply:
pitti liked the new name better.  I agree :)